packages/build/src/error/types.ts

```diff
@@ -449,6 +466,24 @@ const TYPES: { [T in ErrorTypes]: ErrorType } = {
     locationType: 'api',
     severity: 'error',
   },
+  /**
+   * Non-internal errors deploying files or functions
+   */
+  deploy: {
+    title: 'Error deploying',
+    stackType: 'none',
+    locationType: 'deploy',
+    severity: 'info',
+  },
+  /**
+   * Internal errors deploying files or functions
+   */
+  deployInternal: {
+    title: 'Internal error deploying',
+    stackType: 'none',
+    locationType: 'deploy',
+    severity: 'error',
+  },
```

Comment:
New set of errors:
- Deploy should encompasses any non 5xx
- Deploy internal any 5xx.

The main distinction being that we see one as internal system error and the other as a use error.